### PR TITLE
SONARIAC-1205 S6868: Allowing command execution is security sensitive

### PR DESCRIPTION
**Pull request by jonas-wielage-sonarsource**

### CODE REVIEW

File: iac-extensions/kubernetes/src/test/resources/checks/CpuLimitCheck/test_template_object.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ spec:
         # ^^^^
           resources:
             requests:
-              memory: 64Mi
+              cpu: 64Mi
```
Comment (petertrr) [inline iac-extensions/kubernetes/src/test/resources/checks/CpuLimitCheck/test_template_object.yaml:28]: Why is this changed? Is `Mi` a valid quantifier for CPU?

File: iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/checks/CommandExecutionCheck.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  private static boolean ruleContainsSensitiveVerb(BlockObject rule) {
+    return containsSensitiveItem(rule, "verbs", "create");
+  }
+
+  private static boolean ruleContainsSensitiveResource(BlockObject rule) {
+    return containsSensitiveItem(rule, "resources", "pods/exec");
+  }
+
+  private static boolean containsSensitiveItem(BlockObject rule, String listKey, String sensitiveItem) {
```
Comment (petertrr) [inline iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/checks/CommandExecutionCheck.java:57]:
```suggestion
  private static boolean containsSensitiveItemOrWildcard(BlockObject rule, String listKey, String sensitiveItem) {
```

**Review by petertrr — APPROVED** [review]
LGTM

File: iac-extensions/kubernetes/src/test/resources/checks/CpuLimitCheck/test_template_object.yaml (hunk 1 above)
Comment (jonas-wielage-sonarsource) [reply iac-extensions/kubernetes/src/test/resources/checks/CpuLimitCheck/test_template_object.yaml:28]: Good catch, the `Mi` should be removed. I changed it here, because I remarked this on another PR and it was missed to change.
